src/dbus/server/dbus_agent.cpp

```diff
@@ -91,6 +98,8 @@ void DBusAgent::Update(MainloopContext &aMainloop)
     unsigned int flags;
     int          fd;
 
+    VerifyOrExit(mConnection != nullptr);
```

Comment:
should we simply crash if dbus connection failed?

Reply:
Done.